view_w.php

```diff
@@ -39,7 +39,7 @@
 $nextStr = translate ( 'Next' );
 $prevStr = translate ( 'Previous' );
 
-$can_add = ( empty ( $ADD_LINK_IN_VIEWS ) || $ADD_LINK_IN_VIEWS != 'N' );
+$can_add = ( $ADD_LINK_IN_VIEWS === 'Y' );
```

Comment:
I've made so many changes, I lose track of a few. Although, I'm getting better at it.
How would I remove that change? Which pull request is it related to?
For that matter, is there a way to cancel a pull request altogether?